blockchain/v2/reporter.go

```diff
@@ -7,13 +7,13 @@ import (
 	"github.com/tendermint/tendermint/p2p"
 )
 
-// Reporter provides an interface for reactors to report the behaviour
+// Reporter provides an interface for reactors to report the behavior
 // of peers synchronously to other components.
 type Reporter interface {
```

Comment:
These are still publicly exposed interfaces. Should I make them private?

Reply:
If they're moved into internal, it won't matter I believe.

Reply:
ah right but it isn't in an `internal` folder. I've just moved it to be a part of the blockchain/v2 package

Reply:
It seems we have two options (assuming they're only used within blockchain v2):

1. Use an `internal` package and keep it exported
2. Make these types private

Reply:
Yup. What's your preference?

Reply:
I have none, but keep in mind:

Option (1):
- leans towards the approach we're ultimately going to take towards Tendermint 1.0, so it's more "idiomatic"
- the PR diff will be larger, but I guess this isn't a big deal

Option (2):
- probably less work for you
- smaller PR diff

Reply:
Let's do Option 1, but as a separate PR from all these spelling changes. I can revert the linter change if it's causing trouble between now and then.

Reply:
Cool shall I revert the last couple of commits and we get this in as the lint pleaser PR and then I will open a new one internalizing the behavior package

Reply:
Yep. I'll remove the linter in a separate PR as well

Reply:
I don't think you will need to